packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -464,6 +472,60 @@ export class StripeWebhookHandler extends StripeHandler {
     await this.sendSubscriptionPaymentFailedEmail(invoice);
   }
 
+  /**
+   * Handle `invoice.upcoming` Stripe wehbook events.
+   */
+  async handleInvoiceUpcomingEvent(request: AuthRequest, event: Stripe.Event) {
+    const invoice = event.data.object as Stripe.Invoice;
+    if (invoice.billing_reason !== 'upcoming') {
+      // Send payment failure emails only when processing a subscription renewal.
```

Comment:
Payment failure email?

Reply:
Given the name of the webhook event is this possible?

Reply:
Yup that's right. I've removed this check.